Dockerfile

```diff
@@ -4,20 +4,34 @@
 
 # Use bullseye as build image instead of Bookworm as ubi9 does not not have GLIBCXX_3.4.30
 # https://access.redhat.com/solutions/6969351
-FROM --platform=${BUILDPLATFORM} rust:1.78.0-bullseye as limitador-build
```

Comment:
is this breaking cross platform builds?

Reply:
why would it? This Dockerfile works fine on my arm mac.

Reply:
https://docs.docker.com/reference/dockerfile/#from

```
The optional --platform flag can be used to specify the platform of the image in case FROM references a multi-platform image. 
For example, linux/amd64, linux/arm64, or windows/amd64. By default, the target platform of the build request is used. Global build arguments can be used in the value of this flag, for example [automatic platform ARGs](https://docs.docker.com/reference/dockerfile/#automatic-platform-args-in-the-global-scope) 
allow you to force a stage to native build platform (--platform=$BUILDPLATFORM), and use it to cross-compile to the target platform inside the stage.
```

That was there for a reason and now you are removing it. Is there a reason for that or the optional `--platform=${BUILDPLATFORM}` was useless?

Reply:
Yeah, I think it's best if the plain Dockerfile just does a container build that matches the host, and then we have additional Dockerfiles to do the cross compiles like [Dockerfile.aarch64](https://github.com/Kuadrant/limitador/blob/main/Dockerfile.aarch64).